distribution/changelog.txt

```diff
@@ -1,4 +1,4 @@
-0.2.4+ (in development)
+﻿0.2.4+ (in development)
```

Comment:
It looks like your editor (probably one created by Microsoft) inserted a Byte Order Mark here. Please remove it.